service/resource/endpoint/create.go

```diff
@@ -89,5 +50,15 @@ func (r *Resource) newCreateChange(ctx context.Context, obj, currentState, desir
 		}
 	}
 
-	return createState, nil
+	if len(createState.IPs) == 0 {
+		// Nothing to do.
+		return nil, nil
```

Comment:
This is the case in which `ApplyCreateChange` will probably fail.

Reply:
Should be fixed now.